lib/libp2p_port.ex

```diff
@@ -299,6 +293,13 @@ defmodule LambdaEthereumConsensus.Libp2pPort do
     OperationsCollector.init()
   end
 
+  @spec enable_request_handlers(port()) :: :ok | {:error, String.t()}
+  defp enable_request_handlers(port) do
+    IncomingRequestsHandler.protocol_ids()
+    |> Stream.map(fn protocol_id -> set_handler(protocol_id, port) end)
+    |> Enum.each(fn true -> nil end)
```

Comment:
Why not an Enum.each directly?

```suggestion
    IncomingRequestsHandler.protocol_ids()
    |> Enum.each(fn protocol_id -> set_handler(protocol_id, port) end)
```

Alternatively, if the goal is to make it fail explicitely we can 

```elixir
    IncomingRequestsHandler.protocol_ids()
    |> Enum.each(fn protocol_id -> true = set_handler(protocol_id, port) end)
```